src/Controller/Api/UserController.php

```diff
@@ -96,4 +99,83 @@ public function userListAction(Request $request, UserListViewFactory $factory)
 
         return $factory->create($userCollection);
     }
+
+    /**
+     * @SWG\Tag(name="Keys")
+     *
+     * @SWG\Response(
+     *     response=200,
+     *     description="List of user keys",
+     *     @Model(type="\App\Model\View\User\UserKeysView")
+     * )
+     * @SWG\Response(
+     *     response=204,
+     *     description="User has no keys"
+     * )
+     * @SWG\Response(
+     *     response=401,
+     *     description="Unauthorized"
+     * )
+     *
+     * @Route(
+     *     path="/api/keys",
+     *     name="api_keys_list",
+     *     methods={"GET"}
+     * )
+     *
+     * @param UserKeysViewFactory $viewFactory
+     *
+     * @return UserKeysView|null
+     */
+    public function keyListAction(UserKeysViewFactory $viewFactory)
```

Comment:
It may be better to extend the SelfUserInfoView class and use \App\Controller\Api\UserController::userInfoAction